curvefs/src/tools/status/curvefs_status_base_tool.h

```diff
@@ -69,7 +69,7 @@ class StatusBaseTool : public CurvefsToolMetric {
     std::string version_;
     std::string hostType_;
     std::string versionSubUri_ = "";
-    std::string StatusSubUri_ = "";
+    std::string statusSubUri_ = "";
```

Comment:
why some members have a default value and other not?

Reply:
> why some members have a default value and other not?

uri has deafult value, if you don't want get this value set it to deafult value.